jdk/make/closed/autoconf/custom-spec.gmk.in

```diff
@@ -103,6 +103,12 @@ ifeq ($(OPENJDK_BUILD_OS), macosx)
   LDFLAGS_JDKEXE += -pagezero_size 0x1000
 endif
 
+ifeq ($(OPENJDK_BUILD_OS), windows)
```

Comment:
Technically, this (and the macosx test above) should check `OPENJDK_TARGET_OS` instead of `OPENJDK_BUILD_OS`.